packages/playwright/src/isomorphic/testServerInterface.ts

```diff
@@ -81,13 +81,15 @@ export interface TestServerInterface {
     locations?: string[];
     grep?: string;
     grepInvert?: string;
+    onlyChanged?: string;
```

Comment:
Let's make `onlyChanged`-related changes in a separate PR? IIUC, we do not support `onlyChanged` in UI mode as well?

Reply:
Happy to move it into a different PR if you want. The reason I included it here is that we supported onlyChanged in the previous impl of watch mode, and had a test for it: https://github.com/Skn0tt/playwright/blob/e8ad5cac6f9071293297287053c4c5313c65b1d2/tests/playwright-test/only-changed.spec.ts#L169